styles.css

```diff
@@ -0,0 +1,46 @@
+.container{
+    width: min(95%,1200px);
+    margin: 0 auto;
+    background-color: rgb(138, 196, 247);
+}
+
+#title, #description{
+    text-align: center;    
+}
+
+#description{
+    margin-bottom: 51px;
+}
+
+.input-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.input-description{
+    margin-bottom: 10px;
+}
+
+.center{
+    text-align: center;
+}
+
+.radioButton, .checkboxContainer{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.commentsP{
+    text-align: center;
+}
+
+#comments{
+    margin-top: 20px;
+}
+
+.buttonContainer,.selectContainer{
+    display: flex;    
+    justify-content: center;
+}
```

Comment:
Here you will have some linters error, It will be great, if you correct them

Reply:
- [ ] Remember that linters errors are a reason to invalidate a code review because linters are a general requirement for each project. I would suggest invalidating a project that is not passing instead of reviewing it. Linters are like the first check before a code review :pushpin: :wink:
- [ ] [OPTIONAL] I think it could be a good idea to add emojis to your comments to make the communication more human :hugs: